src/libs/migrations/CheckForPreviousReportActionID.ts

```diff
@@ -44,24 +40,25 @@ export default function () {
             }),
         );
 
-        if (_.isUndefined(firstValidValue)) {
+        if (!firstValidValue) {
             Log.info(`[Migrate Onyx] Skipped migration CheckForPreviousReportActionID because there were no valid reportActions`);
             return;
         }
 
-        if (_.has(firstValidValue, 'previousReportActionID')) {
+        if (firstValidValue.previousReportActionID) {
             Log.info(`[Migrate Onyx] CheckForPreviousReportActionID Migration: previousReportActionID found. Migration complete`);
             return;
         }
 
         // If previousReportActionID not found:
         Log.info(`[Migrate Onyx] CheckForPreviousReportActionID Migration: removing all reportActions because previousReportActionID not found in the first valid reportAction`);
 
-        const onyxData = {};
-        _.each(allReportActions, (reportAction, onyxKey) => {
+        const onyxData: OnyxCollection<OnyxTypes.ReportActions> = {};
+
+        Object.keys(allReportActions ?? {}).forEach((onyxKey) => {
             onyxData[onyxKey] = {};
         });
 
-        return Onyx.multiSet(onyxData);
+        return Onyx.multiSet(onyxData as Record<`${typeof ONYXKEYS.COLLECTION.REPORT_ACTIONS}`, Record<string, never>>);
```

Comment:
@fvlvte Maybe we can update the `Onyx.multiSet` type to get rid of the assertion?